templates/example-plans/020-StandardInfraConfig-default-with-mqtt.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       storage: 2Gi
     addressFullPolicy: FAIL
   router:
-    minReplicas: 2
+    minReplicas: 3
```

Comment:
This seems like an unrelated change. The minReplicas of routers does not affect queue partitions. The partitions setting will only scale up the broker, which is handled by the operator.

Reply:
It is, was easier to include it in this PR but should really be a separate PR. would you like me to split it out?

Reply:
what's the rationale for having a different config with a minimum router replica of three anyway... why not 4, 5, 17... etc

Reply:
I don’t see the point in increasing minreplicas beyond 2 @davidffrench

Reply:
These are good points, 2 routers are sufficient for HA in these sample plans. I have removed this commit.